src/all/unionmangas/src/eu/kanade/tachiyomi/extension/all/unionmangas/UnionMangasDto.kt

```diff
@@ -116,7 +116,7 @@ class MangaDetailsDto(
 
     val thumbnailUrl get() = "${UnionMangas.apiUrl}$_thumbnailUrl"
     val genres get() = _genres.joinToString { it.name }
-    val status get() = toSMangaStatus(_status.first().name)
+    val status get() = if (_status.isNotEmpty()) toSMangaStatus(_status.first().name) else SManga.UNKNOWN
```

Comment:
can do `_status.firstOrNull() ?: ""` which would make the function return unknown anyway